docs/core/tutorials/top-level-templates.md

```diff
@@ -16,14 +16,12 @@ The new output uses recent C# features that simplify the code you need to write
 
 ```csharp
 using System;
-using System.Collections.Generic;
-using System.Linq;
 
 namespace MyApp // Note: actual namespace depends on the project name.
 {
-    public class Program
+    class Program
```

Comment:
```suggestion
    internal class Program
```

Reply:
@tdykstra The template itself doesn't contain "internal" (tested from command-line). The behavior in Visual Studio is an IDE feature. But since the majority of users are using Visual Studio, I'm adding the accessibility, especially that it doesn't change the semantics of the code.